.github/workflows/codeql-analysis.yml

```diff
@@ -0,0 +1,65 @@
+# For most projects, this workflow file will not need changing; you simply need
```

Comment:
I wonder if we can add this to CI or something?

Reply:
I'm pretty sure adding this file here, in the `.github/workflows` folder, adds it to Github Actions.

Reply:
Correct, this change makes this live on GitHub Actions.

Reply:
Yeah, I just meant that i'd be rad if we could include this somewhere for all repos so we don't have to configure it in github everywhere 🤷‍♀️